apps/qubit/modules/default/templates/_googleAnalytics.php

```diff
@@ -1,4 +1,4 @@
-<?php $gaKey = sfConfig::get('app_google_analytics_api_key', ''); ?>
+<?php $gaKey = '' != QubitSetting::getByName('google_analytics') ? QubitSetting::getByName('google_analytics') : sfConfig::get('app_google_analytics_api_key', ''); ?>
```

Comment:
Avoid retrieving the setting twice:
```suggestion
<?php
$gaKey = QubitSetting::getByName('google_analytics');
if (empty($gaKey)) {
    $gaKey = sfConfig::get('app_google_analytics_api_key', '');
}
?>
```